### ISSUE TITLE
dupe

### ISSUE THREAD

**Issue by SrMonsterYT** [BG-Software-LLC/WildChests#240]
### Minecraft's Version

 git-Pufferfish-46 (MC: 1.20.4)

### Plugin's Version

WildChests-2024.1-b49.jar

### Describe the bug

I don't know how this happened, but the wildchests are bugged and it's possible to duplicate them, so now I've updated it to WildChests-2024.1-b53.jar.

### To Reproduce

[video](bug: https://youtu.be/4fAoyTiUcJs)

**Comment by OmerBenGera** [comment]
Let me know how to replicate the issue.

**Comment by SrMonsterYT** [comment]
> Let me know how to replicate the issue.

I don't know how this happened, but some players reported that the items in the chest became invisible, so they put an item inside and this happened, we don't know yet, I believe it's after the server restart the chests were not updated correctly?

**Comment by OmerBenGera** [comment]
> > Let me know how to replicate the issue.
> 
> I don't know how this happened, but some players reported that the items in the chest became invisible, so they put an item inside and this happened, we don't know yet, I believe it's after the server restart the chests were not updated correctly?

Try replicating it on another server or something, look for suspicious things in the db...

**Comment by OmerBenGera** [comment]
> > > > Let me know how to replicate the issue.
> > > 
> > > 
> > > I don't know how this happened, but some players reported that the items in the chest became invisible, so they put an item inside and this happened, we don't know yet, I believe it's after the server restart the chests were not updated correctly?
> > 
> > 
> > Try replicating it on another server or something, look for suspicious things in the db...
> 
> I believe it happened when the server simply restarted without warning, it just shut down without errors and logs

Hmm?

**Comment by SrMonsterYT** [comment]
> > > > > Let me know how to replicate the issue.
> > > > 
> > > > 
> > > > I don't know how this happened, but some players reported that the items in the chest became invisible, so they put an item inside and this happened, we don't know yet, I believe it's after the server restart the chests were not updated correctly?
> > > 
> > > 
> > > Try replicating it on another server or something, look for suspicious things in the db...
> > 
> > 
> > I believe it happened when the server simply restarted without warning, it just shut down without errors and logs
> 
> Hmm?

I discovered the problem, the server was on 1.20.4, when I updated to 1.20.6 it got this bug again, the only solution is to delete the database and all players lose their wildchest

video: https://youtu.be/GKVb89BgJzo

the solution is always to delete database.db every time you update the server

**Comment by OmerBenGera** [comment]
> > > > > > Let me know how to replicate the issue.
> > > > > 
> > > > > 
> > > > > I don't know how this happened, but some players reported that the items in the chest became invisible, so they put an item inside and this happened, we don't know yet, I believe it's after the server restart the chests were not updated correctly?
> > > > 
> > > > 
> > > > Try replicating it on another server or something, look for suspicious things in the db...
> > > 
> > > 
> > > I believe it happened when the server simply restarted without warning, it just shut down without errors and logs
> > 
> > 
> > Hmm?
> 
> I discovered the problem, the server was on 1.20.4, when I updated to 1.20.6 it got this bug again, the only solution is to delete the database and all players lose their wildchest
> 
> video: https://youtu.be/GKVb89BgJzo
> 
> the solution is always to delete database.db every time you update the server

I can't seem to replicate the issue
Are you able to replicate it on a standalone server?

**Comment by OmerBenGera** [comment]
Any news regarding this?

**Comment by SrMonsterYT** [comment]
> Any news regarding this?

this happens every time I update the server to another version, then it happens, on the test server I couldn't replicate only on the production one.

**Comment by OmerBenGera** [comment]
> > Any news regarding this?
> 
> this happens every time I update the server to another version, then it happens, on the test server I couldn't replicate only on the production one.

Take the DB and config file of WC to your test server, and prepare a chest at a location of a random storage unit and check if it happens.
If you struggle with it, send it to me and I'll try to do it

**Comment by SrMonsterYT** [comment]
> > > Any news regarding this?
> > 
> > 
> > this happens every time I update the server to another version, then it happens, on the test server I couldn't replicate only on the production one.
> 
> Take the DB and config file of WC to your test server, and prepare a chest at a location of a random storage unit and check if it happens. If you struggle with it, send it to me and I'll try to do it

only the chests that are in the world are like this, if I take 1 new chest and put it on the ground it stays normal, as I said, this happened when I updated from 1.20.4, to 1.20.6

**Comment by OmerBenGera** [comment]
Unless you can't replicate it or give me at least more information about the bug, I can't do much.

**Comment by OmerBenGera** [comment]
Does it only happen if you upgrade between 1.20.4 and 1.20.6?

**Comment by SrMonsterYT** [comment]
> Does it only happen if you upgrade between 1.20.4 and 1.20.6?

yes, it happened when I updated to 1.20.1 to 1.20.4, then 1.20.4 to 1.20.6

**Comment by OmerBenGera** [comment]
Can't seem to replicate the issue
You might have used an old version of WildChests that had issues when upgrading from 1.20.4 to 1.20.6 but it was fixed long time ago

**Comment by LoneDevilX** [comment]
hi there, i foudn another dupe not sure if its the same case, suction chest:

Error:
https://pastebin.com/Pj2ezXy0

Config:
https://pastebin.com/nRuu8Lrw

Video:
https://github.com/user-attachments/assets/2bb9c882-e42d-4ea0-8792-e420fc395990

**Comment by OmerBenGera** [comment]
> hi there, i foudn another dupe not sure if its the same case, suction chest:
> 
> Error: https://pastebin.com/Pj2ezXy0
> 
> Config: https://pastebin.com/nRuu8Lrw
> 
> Video: https://github.com/user-attachments/assets/2bb9c882-e42d-4ea0-8792-e420fc395990

Update to latest dev build
If the issue persists, open a new bug report - this is not related to the one reported in here

**Comment by OmerBenGera** [comment]
Closing this for now
If the issue continues, lmk
